.pre-commit-config.yaml

```diff
@@ -42,7 +42,7 @@ repos:
       - id: black
 
   - repo: https://github.com/PyCQA/flake8
-    rev: 5.0.4
+    rev: 6.0.0
```

Comment:
Did you run `pre-commit autoupdate`?

Reply:
No, I updated this manually

Reply:
should we maybe directly update all used pre-commit packages to the latest version and adapt accordingly?